pkg/config/application.go

```diff
@@ -396,6 +402,12 @@ type WaitApprovalStageOptions struct {
 	MinApproverNum int      `json:"minApproverNum" default:"1"`
 }
 
+type CustomSyncOptions struct {
+	Timeout Duration          `json:"timeout"`
+	Env     map[string]string `json:"env"`
```

Comment:
Should be `envs`? 🤔

Reply:
I name it the same name as Github Actions.
https://docs.github.com/ja/actions/learn-github-actions/variables
and CircleCI uses `environment`
https://circleci.com/docs/ja/env-vars/

Reply:
I guess we should keep the same convention for all stubs, and we have `runs` as a list of commands (plural), wdyt?

Reply:
I thought `env` is not list but map, so it does not have to be plural, but I saw other map field uses plural https://pipecd.dev/docs/user-guide/configuration-reference/
I think `envs` is better in this case, too.

Reply:
OKay, lets use `envs` to keep unify with others configurations in pipecd.